Packs/ContentTesting/Scripts/UnitTestPlaybookAnalyzer/UnitTestPlaybookAnalyzer.yml

```
@@ -1,17 +1,27 @@
 args:
-- description: Name of the playbook to analyze
+- description: Name of the playbook to analyze, a value must be provided.
   name: playbook
-- description: Limit incidents to the most recent since "N days ago" to match playbook implementation
-  name: occurred
+  required: false
+- description: Sub-playbook for analysis versus the parent playbook
+  name: subplaybook
+- description: 'First day to find incident occurrences using the playbook. A value must be provided, for example: "2023-03-01".'
+  name: firstday
+  required: false
+- description: 'Last day to find incident occurrences using the playbook: Example: "2023-03-31"'
+  name: lastday
   required: true
```

Notice, this is breaking changes- therefore deleting this argument would cause a failure for customers already using this pack. Would you consider keeping it and adding "hide"? 

```suggestion
  required: false
  hide: true
```